lib/ReactGridLayout.jsx

```diff
@@ -136,6 +137,8 @@ export default class ReactGridLayout extends React.Component<Props, State> {
     draggableCancel: PropTypes.string,
     // A selector for the draggable handler
     draggableHandle: PropTypes.string,
+    // A dataTransfer type that is allowed to drop on the grid.
+    allowedDropTypes: PropTypes.string,
```

Comment:
I forget, if you pass in `allowedDropTypes: null` does it complain? Or is this valid as a type of string?

Reply:
Good shout, I don't think it will complain because it isn't a required prop, so null is accepted. I'll update to do a null check later in the code